live-examples/css-examples/transforms/scale.html

```diff
@@ -0,0 +1,42 @@
+<section id="example-choice-list" class="example-choice-list large" data-property="scale">
+    <div class="example-choice" initial-choice="true">
+        <pre><code class="language-css">scale: 1.5;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">scale: 1.7 0.5;</code></pre>
```

Comment:
Maybe we can add a percentage value for illustration:

```suggestion
        <pre><code class="language-css">scale: 1.7 60%;</code></pre>
```